doc/source/tune/tutorials/tune_get_data_in_and_out.md

```diff
@@ -71,7 +71,7 @@ For example, passing in a large pandas DataFrame or an unserializable model obje
 Instead, use strings or other identifiers as your values, and initialize/load the objects inside your Trainable directly depending on those.
 
 ```{note}
-[Datasets](data_key_concepts) can be used as values in the search space directly.
+[Dataset](data_quickstart) can be used as values in the search space directly.
```

Comment:
Think it should be \[Dataset**s**\]\(data_quickstart\).

Reply:
Updated.